src/lib/salesforceAuthenticator.js

```diff
@@ -20,8 +21,38 @@ export async function authenticate (config: Config) {
     get: function (endpoint: string) {
       return request.get({ uri: `${j.instance_url}${endpoint}`, headers: { 'Authorization': `Bearer ${j.access_token}` } })
     },
+    post: function (endpoint: string, form: any) {
+      return new Promise((resolve, reject) => {
+        request.post({
+          uri: `${j.instance_url}${endpoint}`,
+          headers: {
+            'Authorization': `Bearer ${j.access_token}`
+          },
+          formData: form
+        }, function (err, httpResponse, body) {
+          if (err != null) {
+            console.error(err)
+            reject(err)
+          } else {
+            resolve(body)
+          }
+        })
+      })
+    },
+    url: j.instance_url,
+    auth: `Bearer ${j.access_token}`,
     getp: async function (endpoint: string) {
       return rp.get({ uri: `${j.instance_url}${endpoint}`, headers: { 'Authorization': `Bearer ${j.access_token}` } })
+    },
+    getFulfilmentFolder: async function () {
```

Comment:
Yeah, I want to refactor this file. Currently methods on ES6 classes can't be marked as async.